buildSrc/version.properties

```diff
@@ -28,7 +28,8 @@ joda              = 2.10.2
 #  - distribution/tools/plugin-cli
 #  - x-pack/plugin/security
 bouncycastle      = 1.61
-
+# When updating this version, please review all other SAML dependencies in x-pack/plugin/security
+opensaml          = 3.4.3
```

Comment:
Why do we need this in the global versions? I think it can be local to the security project?